edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -165,10 +190,40 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                         string moduleGenerationId = configuration.GetValue<string>(Constants.EdgeletModuleGenerationIdVariableName);
                         apiVersion = configuration.GetValue<string>(Constants.EdgeletApiVersionVariableName);
                         TimeSpan performanceMetricsUpdateFrequency = configuration.GetTimeSpan("PerformanceMetricsUpdateFrequency", TimeSpan.FromMinutes(5));
-                        builder.RegisterModule(new AgentModule(maxRestartCount, intensiveCareTime, coolOffTimeUnitInSeconds, usePersistentStorage, storagePath, Option.Some(new Uri(workloadUri)), Option.Some(apiVersion), moduleId, Option.Some(moduleGenerationId), enableNonPersistentStorageBackup, storageBackupPath, storageTotalMaxWalSize, storageLogLevel));
-                        builder.RegisterModule(new EdgeletModule(iothubHostname, edgeDeviceHostName, deviceId, new Uri(managementUri), new Uri(workloadUri), apiVersion, dockerAuthConfig, upstreamProtocol, proxy, productInfo, closeOnIdleTimeout, idleTimeout, performanceMetricsUpdateFrequency, useServerHeartbeat, backupConfigFilePath));
+                        builder.RegisterModule(
+                            new AgentModule(
+                                maxRestartCount,
+                                intensiveCareTime,
+                                coolOffTimeUnitInSeconds,
+                                usePersistentStorage,
+                                storagePath,
+                                Option.Some(new Uri(workloadUri)),
+                                Option.Some(apiVersion),
+                                moduleId,
+                                Option.Some(moduleGenerationId),
+                                enableNonPersistentStorageBackup,
+                                storageBackupPath,
+                                storageTotalMaxWalSize,
+                                storageLogLevel));
+                        builder.RegisterModule(
+                            new EdgeletModule(
```

Comment:
seeing some weird tab alignment here

Reply:
checked.  it looks good in Visual Studio.